spec/features/saml/saml_spec.rb

```diff
@@ -167,6 +167,18 @@
   end
 
   context 'visiting /api/saml/logout' do
+    context 'via SP-initiated logout when not logged in to IdP' do
+      it 'does not raise an exception' do
+        request = OneLogin::RubySaml::Logoutrequest.new
+        settings = sp1_saml_settings
+        settings.name_identifier_value = SecureRandom.uuid
+
+        expect do
+          visit request.create(settings)
+        end.to_not raise_error
+      end
+    end
```

Comment:
What I'm saying is that this PR doesn't attempt to address the issue in https://github.com/18F/identity-private/issues/903 and doesn't claim to. All it claims to do is solve a problem with an exception being thrown. If that's not enough, then I'll just close and we can solve the exception when we solve https://github.com/18F/identity-private/issues/903